ext/project_data.yaml

```diff
@@ -19,6 +19,7 @@ gem_required_rubygems_version: '> 1.3.1'
 gem_runtime_dependencies:
   facter: ['> 2.0', '< 4']
   hiera: ['>= 2.0', '< 4']
+  # TODO: but what about this?
```

Comment:
Do we want to set a range on ship? (is there anything useful in 2.0+ of json_pure that we even care about?)